odxtools/basevariantpattern.py

```diff
@@ -0,0 +1,38 @@
+# SPDX-License-Identifier: MIT
+from dataclasses import dataclass
+from typing import List, Union
+from xml.etree import ElementTree
+
+from typing_extensions import override
+
+from .exceptions import odxassert
+from .matchingbasevariantparameter import MatchingBaseVariantParameter
+from .matchingparameter import MatchingParameter
+from .odxlink import OdxDocFragment
+from .variantpattern import VariantPattern
+
+
+@dataclass
+class BaseVariantPattern(VariantPattern):
+    """Base variant patterns are variant patterns used to identify the
+    base variant of an ECU.
+    """
+    matching_base_variant_parameters: List[MatchingBaseVariantParameter]
+
+    @override
+    def get_matching_parameters(
```

Comment:
Why a getter? it takes no arguments

Reply:
because `matching_parameters` conflicts with the `MATCHING-PARAMETERS` dataclass field of ECU variant patterns...

Reply:
(better solution appreciated)

Reply:
You already have `matching_base_variant_parameters`